src/OpenSage.Benchmarks.DataStructures/BenchQuadtreeItem.cs

```diff
@@ -27,11 +27,10 @@ public BenchQuadtreeItem(int id, RectangleF bounds)
 
         public static BenchQuadtreeItem Generate(int id, RectangleF bounds, SizeF maxSize, Random random)
         {
-            var x = (float) random.NextDouble() * bounds.Width - bounds.Left;
-            var y = (float) random.NextDouble() * bounds.Height - bounds.Top;
-
-            var width = MathF.Min(bounds.Right - x, (float) (random.NextDouble() * maxSize.Width));
-            var height = MathF.Min(bounds.Bottom - y, (float) (random.NextDouble() * maxSize.Height));
+            var x = (float)(bounds.X + random.NextDouble() * bounds.Width);
+            var y = (float)(bounds.Y + random.NextDouble() * bounds.Height);
+            var width = (float)(random.NextDouble() * Math.Min(bounds.Width - x, maxSize.Width));
+            var height = (float)(random.NextDouble() * Math.Min(bounds.Height - y, maxSize.Height));
```

Comment:
I replaced this (specifically width/height) to attempt to better distribute the sizes of the items generated. Previously this would generate any size and then cap it to max size, which means that a majority of items would be max size.